### DISCUSSION TITLE
URL to silence

### DISCUSSION THREAD

**Discussion by gtirloni** [grafana/grafana#73167] [Alerting]
I would like to have a URL that I can share with people that points to a specific silence rule.

As is today, I have to point them to the `Alerting > Silences` page and they have to scan it and find the silence.

The edit URL is like this: https://grafana/alerting/silence/a4484b9c-0ea9-4384-980a-6146e9784be1/edit?alertmanager=grafana

But it seems there isn't a way to link directly to a silence. Ideally, it should be possible to right-click and copy the URL to a silence.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82552. Feel free to continue the discussion around this there. Thank you!
